metric_config_parser/tests/test_config.py

```diff
@@ -18,7 +18,7 @@
 from metric_config_parser.outcome import OutcomeSpec
 
 TEST_DIR = Path(__file__).parent
-DEFAULT_METRICS_CONFIG = TEST_DIR / "data" / "default_metrics.toml"
+DEFAULT_METRICS_CONFIG = TEST_DIR / "data" / "jetstream" / "defaults" / "firefox_desktop.toml"
```

Comment:
Doesn't look like this is actually used in here, could we just remove it (and `TEST_DIR`)?